test/onnx/test_operators.py

```diff
@@ -1013,6 +1020,29 @@ def forward(self, x, y):
 
         unregister_custom_op_symbolic('::embedding', _onnx_opset_version)
 
+    def test_shape_value_map(self):
```

Comment:
Testing on .expect file is important, because it measures whether the exported ONNX graph is what we want, especially for shape inference here. Previously the shape cannot be propagated fully, but now it can.

The assertion does not fail before and after the change for this particular unit test. Both works, so I don't add in test_pytorch_onnx_onnxruntime.py but just in test_operators.py.

This unit test is extracted from a large model. The original large model fails because the shape does not propagate fully. This test (with .expect) shows the shape inference is now propagated.

Reply:
Added `test_shape_value_map` in test_pytorch_onnx_onnxruntime.py